server/adaptors/integrations/__data__/repository/nginx/nginx-1.0.0.json

```diff
@@ -8,6 +8,20 @@
   "labels": ["Observability", "Logs", "Flint S3"],
   "author": "OpenSearch",
   "sourceUrl": "https://github.com/opensearch-project/dashboards-observability/tree/main/server/adaptors/integrations/__data__/repository/nginx/info",
+  "workflows": [
```

Comment:
One design choice is that workflows should be user-defined instead of something we hardcode.